Update README with link to intel 3.0 docs (#14)
vchuravy/MCAnalyzer.jl: README.md

## Notes

> `MCAnalyzer.jl` only supports version 3.0 of `iaca`. The [online documentation for version 3.0](https://software.intel.com/sites/default/files/managed/3d/23/intel-architecture-code-analyzer-3.0-users-guide.pdf) is easily available, and contains a few more detailed explanations that this README.

Reviewer: ```suggestion
`MCAnalyzer.jl` only supports version 3.0 of `iaca`. The [online documentation for version 3.0](https://software.intel.com/sites/default/files/managed/3d/23/intel-architecture-code-analyzer-3.0-users-guide.pdf) is easily available, and contains a more detailed explanation and a few more examples.
```
